charts/karpenter/templates/clusterrole-core.yaml

```diff
@@ -65,12 +63,9 @@ rules:
   - apiGroups: [""]
     resources: ["pods/eviction"]
     verbs: ["create"]
-{{- if .Values.webhook.enabled }}
-  - apiGroups: ["admissionregistration.k8s.io"]
-    resources: ["validatingwebhookconfigurations"]
+  - apiGroups: ["apiextensions.k8s.io"]
+    resources: ["customresourcedefinitions"]
```

Comment:
Confirming: We're confident that all of these permissions are strictly required? Do we know _why_ having these permissions is required? Is it to inject the certificate?

Reply:
I can do a sanity check without the permissions, but yes it needs to inject the cert

Reply:
Did the sanity check, we do need it to update the cert.